### DISCUSSION TITLE
Will WindowsServer2022 be supported?

### DISCUSSION THREAD

**Discussion by hyu-pear** [ImageMagick/ImageMagick#5988] [Help]
### ImageMagick version

7.1

### Operating system, version and so on

WindowsServer2019

### Description

I am planning to update the OS of the server to WindowsServer2022.

Please let me know if you know when ImageMagick will support WindowsServer2022.

**Comment by dlemstra** [comment]
Why do you think this is not supported? ImageMagick should be able to run on WindowsServer2022.

**Reply by hyu-pear** [reply]
I agree. If you use it as an individual, it's fine if it can be "run", but I want to use ImageMagick for some of the functions of the system that is publicly provided, so I want to use a supported version.

”https://imagemagick.org/script/download.php" written it supports up to "Windows Server 2019/2016/2012 (x86 & x64)", but if it also supports Windows Server 2022, please add it.

**Reply by dlemstra** [reply]
The text has been changed.
